src/mockdb.ts

```diff
@@ -138,10 +138,21 @@ export class MockDB {
       []
     );
     let allStories = [story, story2, story3, story4];
+    const currentIDs = {};
```

Comment:
I think having a set of currentIDs makes more sense than having a huge list of booleans like this, but I'm fine leaving it like this in the mockdb.